homework/src/Repository.js

```diff
@@ -16,19 +16,50 @@ class Repository {
     //
     // Replace this comment with your code
     //
+
+    const table = Util.createAndAppend('table', parent);
+    const tr = Util.createAndAppend('tr', table);
+    Util.createAndAppend('td', tr, { 'html': 'Repository  : ', 'class': 'tr' });
+    Util.createAndAppend('td', tr, { 'html': this.repo.name });
+
+
+
+    const tr2 = Util.createAndAppend('tr', table);
+    Util.createAndAppend('td', tr2, { 'html': 'Description : ', 'class': 'tr' });
+    Util.createAndAppend('td', tr2, { 'html': this.repo.description });
+
+    const tr3 = Util.createAndAppend('tr', table);
+    Util.createAndAppend('td', tr3, { 'html': 'Forks : ', 'class': 'tr' });
+    Util.createAndAppend('td', tr3, { 'html': this.repo.forks_count });
+
+
+    const Date1 = new Date(this.repo.updated_at).toLocaleString();
+    const tr4 = Util.createAndAppend('tr', table);
+    Util.createAndAppend('td', tr4, { 'html': 'last update : ', 'class': 'tr' });
+    Util.createAndAppend('td', tr4, { 'html': Date1 });
+
+
   }
```

Comment:
Do not add blank lines before the closing brace. They serve no purpose and unnecessarily stretch out your code vertically.